src/http/client.rs

```diff
@@ -393,8 +392,8 @@ impl Http {
         .await
     }
 
-    /// Creates a private thread channel in the [`GuildChannel`] given its Id.
-    pub async fn create_private_thread(
+    /// Creates a thread channel not attached to a message in the [`GuildChannel`] given its Id.
+    pub async fn create_standalone_thread(
```

Comment:
```suggestion
    pub async fn create_thread(
```

Given that `create_thread_from_message` is self-explanatory, no need to include `standalone` in the method name. Same goes for `{ChannelId,GuildChannel}::create_standalone_thread`.